### ISSUE TITLE
Draw scripts should take position as an argument

### ISSUE THREAD

**Issue by shippy** [YaleDecisionNeuro/PsychTaskFramework#98]

**Comment by shippy** [comment]
The current roadblock to implementing this is that I made it a rule that all draw scripts have the same interface: each takes `(trialData, blockConfig)` as arguments. But it should not be the responsibility of the draw functions to provide that interface, and it doesn't need to be: instead, draw functions should take whatever properties they want, and the common interface should be implemented by wrappers that each task provides. 

To take an example from #26, instead of defining the phase like this:

```matlab
phaseConfig('feedback', 0.5, @phase_generic, @action_display, @drawFeedback)
```

...the feedback drawing function could be written to take arguments for horizontal and vertical positioning like so:

```matlab
phaseConfig('feedback', 0.5, @phase_generic, @action_display, ...
  @(trial, block) drawFeedback(block.device.screenId, 'XPos', 'center', ...
     'YPos', trial.feedbackVerticalPosition, 'choice', trial.choice))
```

If anonymous functions are too confusing, this is equivalent to writing

```matlab
% lib/draw/wrappers/wrapper_drawFeedback.m
function wrapper_drawFeedback(trialData, blockConfig)
  drawFeedback(blockConfig.device.screenId, 'XPos', 'center', ... 
    'YPos', trialData.feedbackVerticalPosition, 'choice', trialData.choice)
end

% ...whatever config file
phaseConfig('feedback', 0.5, @phase_generic, @action_display, @wrapper_drawFeedback)
```

In other words, if the draw function doesn't need to know anything about the trial so far, the wrapper doesn't need to pass it that information. If it does, the wrapper can pass just the relevant information. Either way, this means that the draw function doesn't need to magically know where to locate the right values in the configuration.

As usual, the problem is that this introduces yet another layer of abstraction, which at this point would be... three or four levels deep? This is exactly the kind of a thing that made frameworks difficult for me to adopt. That said, I think this might be necessary for greater reusability and independence from purely risk-and-ambiguity tasks.
